patches/tModLoader/Terraria/ModLoader/Default/ModAccessorySlotPlayer.cs

```diff
@@ -3,40 +3,65 @@
 using System.IO;
 using System.Linq;
 using Terraria.DataStructures;
+using Terraria.ID;
+using Terraria.Localization;
 using Terraria.ModLoader.IO;
 
 namespace Terraria.ModLoader.Default;
 
 // Test in Multiplayer, suspect there is some issue with synchronization of unloaded slots
 public sealed class ModAccessorySlotPlayer : ModPlayer
 {
-	internal static AccessorySlotLoader Loader => LoaderManager.Get<AccessorySlotLoader>();
+	private static AccessorySlotLoader Loader => LoaderManager.Get<AccessorySlotLoader>();
 
 	// Arrays for modded accessory slot save/load/usage. Used in DefaultPlayer.
+	/// <summary> <see cref="ModAccessorySlot"/> corollary to the accessory and vanity slots of <see cref="Player.armor"/>. </summary>
 	internal Item[] exAccessorySlot;
+	/// <summary> <see cref="ModAccessorySlot"/> corollary to <see cref="Player.dye"/>. </summary>
 	internal Item[] exDyesAccessory;
+	/// <summary> <see cref="ModAccessorySlot"/> corollary to <see cref="Player.hideVisibleAccessory"/>. </summary>
 	internal bool[] exHideAccessory;
-	internal Dictionary<string, int> slots = new Dictionary<string, int>();
+	/// <summary> All slots that can potentially show for this user. First the loaded ModAccessorySlots, followed by any unloaded slots. </summary>
+	private readonly Dictionary<string, (int SlotType, bool HasLoadoutSupport)> slots = [];
```

Comment:
Looks like this `bool HasLoadoutSupport` is unused now.

Reply:
Done.